app/models/data_point.rb

```diff
@@ -141,12 +140,14 @@ def DataPoint.delete_all_created_on(date)
   end
 
   def DataPoint.update_public_data_points(universe = 'UHERO', series = nil)
+    remove_quarantine = FeatureToggle.is_set('remove_quarantine_from_public', universe, true)
```

Comment:
This default should be false, since we know that is the normal desired behavior.

Reply:
Oh! I thought `true` was the normal desired behavior.